gulp/js/router.js

```diff
@@ -7,8 +7,8 @@ app.routers.DefaultRouter = Backbone.Router.extend({
     "transcripts/:id?*queryString": "transcriptEdit",
     "page/:id":                     "pageShow",
     "dashboard":                    "dashboard",
-    "search":                       "search",
-    "search?*queryString":          "search"
+    // "search":                       "search",
```

Comment:
On this commented out code and the following is there a reason to keep it here for now?

Reply:
I'll delete it